content/en/docs/concepts/storage/volumes.md

```diff
@@ -62,96 +60,23 @@ a different volume.
-storage driver instead.
-
-### azureFile (deprecated) {#azurefile}
-
-{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
-
-The `azureFile` volume type mounts a Microsoft Azure File volume (SMB 2.1 and 3.0)
-into a pod.
-
-For more details, see the [`azureFile` volume plugin](https://github.com/kubernetes/examples/tree/master/staging/volumes/azure_file/README.md).
-
-#### azureFile CSI migration
-
-{{< feature-state for_k8s_version="v1.26" state="stable" >}}
-
-The `CSIMigration` feature for `azureFile`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `file.csi.azure.com` Container
-Storage Interface (CSI) Driver. In order to use this feature, the [Azure File CSI
-Driver](https://github.com/kubernetes-sigs/azurefile-csi-driver)
-must be installed on the cluster and the `CSIMigrationAzureFile`
-[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled.
-
-Azure File CSI driver does not support using same volume with different fsgroups. If
-`CSIMigrationAzureFile` is enabled, using same volume with different fsgroups won't be supported at all.
-
-#### azureFile CSI migration complete
-
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
-
-To disable the `azureFile` storage plugin from being loaded by the controller manager
-and the kubelet, set the `InTreePluginAzureFileUnregister` flag to `true`.
-
-### cephfs
-{{< feature-state for_k8s_version="v1.28" state="deprecated" >}}
-
-{{< note >}}
-The Kubernetes project suggests that you use the [CephFS CSI](https://github.com/ceph/ceph-csi) third party
-storage driver instead.
-{{< /note >}}
-
-A `cephfs` volume allows an existing CephFS volume to be
-mounted into your Pod. Unlike `emptyDir`, which is erased when a pod is
-removed, the contents of a `cephfs` volume are preserved and the volume is merely
-unmounted. This means that a `cephfs` volume can be pre-populated with data, and
-that data can be shared between pods. The `cephfs` volume can be mounted by multiple
-writers simultaneously.
-
-{{< note >}}
-You must have your own Ceph server running with the share exported before you can use it.
-{{< /note >}}
-
-See the [CephFS example](https://github.com/kubernetes/examples/tree/master/volumes/cephfs/) for more details.
-
-### cinder (removed) {#cinder}
-
-<!-- maintenance note: OK to remove all mention of cinder once the v1.26 release of
-Kubernetes has gone out of support -->
-
-Kubernetes {{< skew currentVersion >}} does not include a `cinder` volume type.
-
-The OpenStack Cinder in-tree storage driver was deprecated in the Kubernetes v1.11 release
-and then removed entirely in the v1.26 release.
-
-The Kubernetes project suggests that you use the 
-[OpenStack Cinder](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md)
-third party storage driver instead.
+- [`awsElasticBlockStore`](/docs/reference/storage/#awselasticblockstore)
```

Comment:
I think this was removed in v1.27 - the page implies the removal was in v1.28

My suggestion - list the removal version inline for each removed legacy in-tree driver.